connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinatorTest.java

```diff
@@ -93,9 +96,9 @@ public class WorkerCoordinatorTest {
     private final ConnectorTaskId taskId3x0 = new ConnectorTaskId(connectorId3, 0);
 
     private final String groupId = "test-group";
-    private final int sessionTimeoutMs = 10;
+    private final int sessionTimeoutMs = 30;
     private final int rebalanceTimeoutMs = 60;
-    private final int heartbeatIntervalMs = 2;
+    private final int heartbeatIntervalMs = 7;
```

Comment:
I increased `sessionTimeoutMs` and `heartbeatIntervalMs` by small numbers so as to be able to mock a poll timeout expiry without a lot of session timeouts and losing connectivity with Group Coordinator.